apprentice_teacher.rb

```diff
@@ -9,10 +13,6 @@ def initialize(options={})
     @phase = 3
   end
 
-  def offer_high_five
-    "High five!"
-  end
-
   def set_phase(num)
```

Comment:
Quite a bit of this is duplicating functionality that the superclass already has!  The whole point of the exercise was to avoid the repeated code.